infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -9,19 +9,26 @@ exports.tasks.push({
     'target-roles',
   ],
   run: async () => {
-    let auth = new taskcluster.Auth(taskcluster.fromEnvVars());
-    let randomId = taskcluster.slugid();
-    let roleId = `project:taskcluster:smoketest:${randomId}`;
-    let payload = {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    const randomId = taskcluster.slugid();
+    const roleId = `project:taskcluster:smoketest:${randomId}*`;
+    const payload = {
       description: 'smoketest test',
-      scopes: ['project:taskcluster:smoketest:*'],
+      scopes: ['project:taskcluster:smoketest:<..>/*'],
     };
-    let expandPayload = {
+    const expandPayload = {
       scopes: [`assume:${roleId}`],
```

Comment:
So, here we'll want to expand `assume:project:taskcluster:smoketest:${randomId}:abc`, and see that the `abc` is substituted properly into the resulting scopes.

Reply:
Yes, I have made the change. Can you take a look please?
@djmitche